### PR TITLE
new: Add help and status icons.

### PR DESCRIPTION
**Pull request by adamirl**
to: @milesj @stefhatcher @hayes @alecklandgraf

## Description

Adding 3 new svg icons to show as help statuses.

## Checklist

Y - My code follows the style guide of this project.
Y - I have updated or added documentation accordingly.
Y - I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by milesj** [comment]
@adamirl Doesnt look like the react components were created/pushed.

File: svgs/interface/call-in.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><circle cx="9" cy="9" r="4"/><path d="M9 15c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4zm7.76-9.64l-1.68 1.69c.84 1.18.84 2.71 0 3.89l1.68 1.69c2.02-2.02 2.02-5.07 0-7.27zM20.07 2l-1.63 1.63c2.77 3.02 2.77 7.56 0 10.74L20.07 16c3.9-3.89 3.91-9.95 0-14z"/><path fill="none" d="M0 0h24v24H0z"/></svg>
```
Comment (stefhatcher) [inline svgs/interface/call-in.svg]: delete `<path fill="none" d="M0 0h24v24H0z"/>` - unused path

File: svgs/interface/phone-msg.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M0 0h24v24H0z" fill="none"/><path d="M20 15.5c-1.25 0-2.45-.2-3.57-.57-.35-.11-.74-.03-1.02.24l-2.2 2.2c-2.83-1.44-5.15-3.75-6.59-6.58l2.2-2.21c.28-.27.36-.66.25-1.01C8.7 6.45 8.5 5.25 8.5 4c0-.55-.45-1-1-1H4c-.55 0-1 .45-1 1 0 9.39 7.61 17 17 17 .55 0 1-.45 1-1v-3.5c0-.55-.45-1-1-1zM12 3v10l3-3h6V3h-9z"/></svg>
```
Comment (stefhatcher) [inline svgs/interface/phone-msg.svg]: delete `<path d="M0 0h24v24H0z" fill="none"/>` - unused path

File: svgs/interface/in-flight.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24"><defs><path id="a" d="M0 0h24v24H0V0z"/></defs><clipPath id="b"><use xlink:href="#a" overflow="visible"/></clipPath><path d="M2.5 19h19v2h-19zm19.57-9.36c-.21-.8-1.04-1.28-1.84-1.06L14.92 10l-6.9-6.43-1.93.51 4.14 7.17-4.97 1.33-1.97-1.54-1.45.39 1.82 3.16.77 1.33 1.6-.43 5.31-1.42 4.35-1.16L21 11.49c.81-.23 1.28-1.05 1.07-1.85z" clip-path="url(#b)"/><path clip-path="url(#b)" fill="none" d="M0 0h24v24H0V0z"/></svg>
```
Comment (stefhatcher) [inline svgs/interface/in-flight.svg]: delete `<defs><path id="a" d="M0 0h24v24H0V0z"/></defs><clipPath id="b"><use xlink:href="#a" overflow="visible"/></clipPath>`

and `clip-path="url(#b)"`

and `<path clip-path="url(#b)" fill="none" d="M0 0h24v24H0V0z"/>`

File: svgs/interface/phone-msg.svg (hunk 2 above)
Comment (stefhatcher) [reply svgs/interface/phone-msg.svg]: after making the updates to the svgs, re-generate the react icons

**Comment by stefhatcher** [comment]
i think these would be more appropriate under "general"

**Comment by adamirl** [comment]
Updated the jank code bits.

File: packages/icons/src/interface/IconCallIn.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import withIcon, { Props } from '../withIcon';
+
+function IconCallIn(props: Props) {
+  return (
+    <svg {...props} viewBox="0 0 24 24">
+      <circle cx="9" cy="9" r="4" />
+      <path d="M9 15c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4zm7.76-9.64l-1.68 1.69c.84 1.18.84 2.71 0 3.89l1.68 1.69c2.02-2.02 2.02-5.07 0-7.27zM20.07 2l-1.63 1.63c2.77 3.02 2.77 7.56 0 10.74L20.07 16c3.9-3.89 3.91-9.95 0-14z" />
+      <path fill="none" d="M0 0h24v24H0z" />
```
Comment (stefhatcher) [inline packages/icons/src/interface/IconCallIn.tsx]: looks like you need to re-generate the react files and run prettier again after that. still seeing the removed paths in these tsx files

File: svgs/interface/in-flight.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24"><path d="M2.5 19h19v2h-19zm19.57-9.36c-.21-.8-1.04-1.28-1.84-1.06L14.92 10l-6.9-6.43-1.93.51 4.14 7.17-4.97 1.33-1.97-1.54-1.45.39 1.82 3.16.77 1.33 1.6-.43 5.31-1.42 4.35-1.16L21 11.49c.81-.23 1.28-1.05 1.07-1.85z"/></svg>
```
Comment (stefhatcher) [inline svgs/interface/in-flight.svg]: `xmlnsXlink="http://www.w3.org/1999/xlink"` doesn't look needed anymore too

**Comment by stefhatcher** [comment]
can you move them under the `general` folder instead of `interface`, please?

**Comment by stefhatcher** [comment]
closing in favor of https://github.com/airbnb/lunar/pull/241

**Pull request closed without merging**